README.md

```diff
@@ -755,6 +755,35 @@ Input:
 
 Output: Same output as for `/api/auth/login` endpoint
 
+
+#### POST /api/auth/register
```

Comment:
@ntnhon  These 2 endpoints + reactivate endpoint should cover the user classic register.